cuv/less.py

```diff
@@ -23,24 +23,20 @@
 
 def term_color(target_fname, cfg, style='monokai'):
     cov = cfg.data
-    match = []
-
-    target_fname = abspath(target_fname)
-    for fname in cov.data.measured_files():
-        if target_fname == abspath(fname):
-            match.append(fname)
-
-    if len(match) != 1:
-        if len(match) == 0:
-            # this file wasn't in the coverage data, so we just dump
-            # it to stdout as-is. (FIXME: ideally, also
-            # syntax-highlighted anyway)
-            with open(target_fname, 'r') as f:
-                for line in f.readlines():
-                    sys.stdout.write(line)
-            return
-        else:
-            raise RuntimeError("Multiple matches: %s" % ', '.join(match))
+    target_fname = realpath(target_fname)
+
+    match = filter( lambda f: target_fname == realpath(f), cov.data.measured_files() )
+
```

Comment:
maybe this should just be a list-comprehension instead; filter() will return a "filter object" in python3, so at least this should be `list(filter(...))` probably. Although probably also many other python3 problems in this codebase ;) so ...meh.

Also, a whole bunch of this "matching" etc. stuff needs to be refactored into a util or similar spot, since it's basically copy-pasta for all the different commands :/ Anyway, something like `[target_fname == realpath(f) for f in cov.data.measured_files()]` ?